movo_7dof_moveit_config/launch/demo.launch

```diff
@@ -29,16 +29,18 @@
   <!-- Run the main MoveIt executable without trajectory execution (we do not have controllers configured by default) -->
```

Comment:
why did u need to make changes to this launch file? it looks like its only launched by robot_demo.launch and sim_demo.launch neither of which we are calling right?